yb-voyager/cmd/exportDataFromTarget.go

```diff
@@ -74,13 +74,18 @@ func initSourceConfFromTargetConf() error {
 	if err != nil {
 		return fmt.Errorf("get migration status record: %v", err)
 	}
+	sourceDBConf := msr.SourceDBConf
 	targetConf := msr.TargetDBConf
 	source.DBType = targetConf.TargetDBType
 	source.Host = targetConf.Host
 	source.Port = targetConf.Port
 	source.User = targetConf.User
 	source.DBName = targetConf.DBName
-	source.Schema = targetConf.Schema
+	if sourceDBConf.DBType == POSTGRESQL {
```

Comment:
What happens if:
- source db has schema = foo
- user has decided to migrate all tables into schema = bar (passed it via --target-db-schema)

We are making an assumption here that it is unchanged.
(was just discussing this with Amit in some other context)...

Reply:
This case is not possible `--target-db-schema` is not supported for PG migration, it is migrated to the same schema as everything is fully-qualified with same schema name while we export using pg_dump. This has to be same as source.

Reply:
`target-db-schema` doesn't work in the case of PG migration, we have this code in `importSchema.go:258`, where we handle this target-db-schema based on source-db-types. 
```
func createTargetSchemas(conn *pgx.Conn) {
	var targetSchemas []string
	tconf.Schema = strings.ToLower(strings.Trim(tconf.Schema, "\"")) //trim case sensitivity quotes if needed, convert to lowercase
	switch sourceDBType {
	case "postgresql": // in case of postgreSQL as source, there can be multiple schemas present in a database
		source = srcdb.Source{DBType: sourceDBType}
		targetSchemas = utils.GetObjectNameListFromReport(analyzeSchemaInternal(), "SCHEMA")
         ....
         
     for _, targetSchema := range targetSchemas {
		//check if target schema exists or not
         ....
	if sourceDBType != POSTGRESQL { // with the new schema list flag, pg_dump takes care of all schema creation DDLs
		schemaExists := checkIfTargetSchemaExists(conn, tconf.Schema)
		createSchemaQuery := fmt.Sprintf("CREATE SCHEMA %s", tconf.Schema)
```